dev-libs/stormlib/stormlib-9.30.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A C++ library for reading, creating, and modifying Blizzard MPQ archives"
+DESCRIPTION="The StormLib library is a pack of modules, written in C++, \
+which are able to read and also to write files from/to the MPQ archives."
+HOMEPAGE="http://www.zezula.net/en/mpq/stormlib.html"
+COPYRIGHT="2003-2024 Ladislav Zezula"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/ladislav-zezula/StormLib/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a709a6b034d206404f5297d85e474371203ff5483639955195d99b737bbf7dfe"
+SOURCE_DIR="StormLib-$portVersion"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="9.22"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Comment:
Not checked, but since it mentions libVersion="9.22"?

```suggestion
libVersionCompat="$libVersion compat >= ${libVersion%.*}"
```

Reply:
If the version contains only 2 parts like here, this doesn't matter. `%` removes only the last part and `%%` only keeps the first, which is the same in this case.